cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -284,7 +302,9 @@ public boolean installAsDependency(VersionRange version, ProcessContext processC
               + " Therefore, we install a compatible version in that range.",
           toolParent, this.tool, version, configuredVersion);
     }
-    ToolInstallation toolInstallation = installTool(version, processContext);
+    ToolInstallation toolInstallation = installTool(
```

Comment:
This change seems to break the `TomcatTest`. Please check why the version determined from the dependency file (21_35) is not being used.